src/coffee/directives/api/models/parent/circle-parent-model.coffee

```diff
@@ -42,43 +42,42 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
           gObject
 
       clean()
-      @listeners = @setEvents gObject, scope, scope, ['radius_changed']
-      if @listeners?
-        @listeners.push google.maps.event.addListener gObject, 'radius_changed', ->
-          ###
-            possible google bug, and or because a circle has two radii
-            radius_changed appears to fire twice (original and new) which is not too helpful
-            therefore we will check for radius changes manually and bail out if nothing has changed
-          ###
+      @listeners = @setEvents(gObject, scope, scope, ['radius_changed']) or [] #or needed incase no-one is listening in controller
```

Comment:
main problem here `@listeners` needs to be set to [] even if scope.events is null